YouTubeClone/YTLiveVideoViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  YTLiveVideoViewController.swift
+//  YouTubeClone
+//
+//  Created by Sagar Dagdu on 29/06/21.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+import AVKit
+import StreamChat
+
+final class YTLiveVideoViewController: UIViewController {
+
+    private let url = URL(string: "https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_adv_example_hevc/master.m3u8")!
+    
+    @IBOutlet private weak var videoView: UIView!
+    @IBOutlet private weak var chatView: UIView!
+    @IBOutlet private weak var playPauseButton: UIButton!
+    
+    @IBOutlet  var videoViewHeightConstraint: NSLayoutConstraint!
+    @IBOutlet  var videoViewFullScreenConstraint: NSLayoutConstraint!
```

Comment:
Do we need an outlet for this constraint? Couldn't find the place where we change it

Reply:
Removed the unused constraint